hack/test-go.sh

```diff
@@ -101,6 +104,11 @@ for arg; do
     testcases+=("${arg}")
   fi
 done
+# If either no test cases were provided, or if the first test case is an empty
+# string (needed to specify Go arguments?), run all tests.
```

Comment:
The empty string case doesn't make sense to me. Can you give me an example of why it would be needed?